content/22.vaccines-trad.md

```diff
@@ -284,7 +280,7 @@ This platform is also favored for its consistency in production and defined comp
 The immune response generated by protein subunit vaccines is weaker, and adjuvants are usually required to boost the response [@doi:10.1517/14712598.2011.573624] (see Appendix).
 These adjuvants are immunogenic substances, which include, for example, alum (aluminum hydroxide), squalene- or saponin-based adjuvants, and Freund’s incomplete/complete adjuvants [@doi:10.1016/j.vaccine.2019.04.055; @doi:10.3390/vaccines2030515].
 
-Table: Approved subunit vaccines [@url:https://covid19.trackvaccines.org/types-of-vaccines] as of {{csse_date_pretty}}
+Table: Subunit vaccines approved for use in at least one country [@url:https://covid19.trackvaccines.org/types-of-vaccines] as of {{csse_date_pretty}} <!--When you say approved, you mean in some countries?-->
```

Comment:
@RLordan thank you for fixing this text! It is the data from VIPER, the criterion is just approval somewhere